.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: Create a bug report to help us improve JustSaying
+
+---
+
+**Describe the bug**
+_A clear and concise description of what the bug is._
+
+**Steps To reproduce**
+_A concise, repeatable, example of how to reproduce the issue._
```

Comment:
Same as my last comment, should this be; _A concise and repeatable example of how to illustrate the issue_?

Reply:
As above.